report/report.tex

```diff
@@ -108,19 +110,39 @@ \section{Problem Formulation}
 %------------------------------------------------
 \section{Methods}
 
-Maecenas sed ultricies felis. Sed imperdiet dictum arcu a egestas.
-\begin{itemize}
-	\item Donec dolor arcu, rutrum id molestie in, viverra sed diam
-	\item Curabitur feugiat
-	\item turpis sed auctor facilisis
-	\item arcu eros accumsan lorem, at posuere mi diam sit amet tortor
-	\item Fusce fermentum, mi sit amet euismod rutrum
-	\item sem lorem molestie diam, iaculis aliquet sapien tortor non nisi
-	\item Pellentesque bibendum pretium aliquet
-\end{itemize}
-\blindtext % Dummy text
+\subsection{Data exploration}
+
+With the cleaned up dataset and the research question in mind*, columns with relevance are explored. To explore, we plot different columns of interest to identify any patterns in the data. We plot the distribution between reviews with votes and no votes as well as review rating.
+\begin{figure}[h]
+	\centering
+	\includegraphics[width=0.5\textwidth]{img/no_votes_vs_votes.pdf}
+	\caption{Vote distribution}
+	\label{fig:votes_distribution}
+\end{figure}
+As seen in \figurename{\ref{fig:votes_distribution}}, the distribution is very uneven in favor of reviews with no votes. This is relevant when evaluating our predictive power. To get another perspective on review votes we plot the data into a heatmap in relation to ratings.
```

Comment:
"This is relevant when evaluating our predictive power" kan måske blive til "This is relevant when evaluating performance metrics of our model, as it indicates that a statistic like Youden's Index might be useful to counteract random guessing".

Reply:
Done